applications/rag/main.tf

```diff
@@ -63,11 +64,13 @@ locals {
 
 ## Generated names for marketplace deployment
 locals {
+  kubernetes_namespace    = var.goog_cm_deployment_name != "" ? "${var.goog_cm_deployment_name}-${var.kubernetes_namespace}" : var.kubernetes_namespace
+  network_name            = var.goog_cm_deployment_name != "" ? "${var.goog_cm_deployment_name}-${var.network_name}" : var.network_name
```

Comment:
Just for my own education, why do we want a new network each time here? Will it cause issues on retry attempts using the same network?

Reply:
yes, for RAG we need cloud SQL with private IP that requires PSA enabled.

currently we cannot check if PSA is already enabled. and hence it it already exists then it will break tf execution.

Hence for smooth experience, new network would smoothen the journey.